go/libraries/utils/filesys/inmemfs.go

```diff
@@ -460,17 +476,20 @@ func (fs *InMemFS) MoveFile(srcPath, destPath string) error {
 			return err
 		}
 
-		destObj := &memFile{destPath, obj.(*memFile).data, destParentDir}
+		now := time.Now()
+		destObj := &memFile{destPath, obj.(*memFile).data, destParentDir, time.Now()}
```

Comment:
Should use now instead of a new call to time.Now()

Reply:
fixed